H/HiGHS/build_tarballs.jl

```diff
@@ -4,7 +4,7 @@ using BinaryBuilder, Pkg
 
 name = "HiGHS"
 
-version = v"1.1.0"
+version = v"1.1.1"
```

Comment:
The source doesn't change?

Reply:
I was just going to rebuild what we had. But I guess it doesn't hurt to bump to the latest `master`. They still aren't doing versioned releases upstream.